src/wizard/wizard.component.jsx

```diff
@@ -17,7 +17,19 @@ export default class Wizard extends React.PureComponent {
   }
 
   componentDidMount() {
-    this.selectPage(undefined, this.props.activeStep);
+    const stepIndex = this.getStepByUrlParam() ? this.getStepByUrlParam() : this.props.activeStep;
```

Comment:
Why call getStepByUrlParam twice?

Reply:
@janekbug: well, I think you can use "this.getStepByUrlParam() || this.props.activeStep" to save space. However, it's not really a costly call in my opinion, just an alternate way of deciding whether we want to use activeStep prop or URL params. As you can see, the first call just checks if there are any params present, and if there is, it uses 'em.